yamale/command_line.py

```diff
@@ -51,7 +51,8 @@ def _find_schema(data_path, schema_name):
     """ Checks if `schema_name` is a valid file, if not
     searches in `data_path` for it. """
 
-    path = glob.glob(schema_name)
+    directory = os.path.dirname(data_path)
```

Comment:
This looks great, thanks! Can we add a test for this so it doesn't break again?

Reply:
Let me poke around with this for a bit, and see what I can do.

Reply:
I have what I think is a reasonable test; I just need to create a clean branch that has only that on it (there was some mucking about and stashish committing going on to get there). I'll try to get that done over the weekend.

The test is to
1. Create `nestedYaml folder` in `command_line_fixtures`
2. Copy `schema.yaml` from `command_line_fixtures` to `nestedYaml`
3. In `nestedYaml`, create a new `good.yaml` and `schema.yaml`, different from all the rest
4. Append the following to `test_command_line.py`:
```
def test_nested_schema_issue_69():
    curdir = os.getcwd()
    os.chdir('yamale/tests/command_line_fixtures/nestedYaml')
    command_line._router('./','schema.yaml', 1, 'PyYAML')
    os.chdir(curdir)
```

If I understand `router` correctly, this should set the same defaults as if `yamale` was called without arguments: It will validate everything from the current working directory on down.

They are on my rather messy [`myTesting` branch](https://github.com/PeterWhittaker/Yamale/tree/myTesting), if you want to peruse (but ignore some of the other stuff, please).

One note: I do not have 3.6, so my successful `tox` tests ran with 3.8 only.

Reply:
And a question: some of the tests explicitly set the number of CPUs to 1, others to 4, some use `@pytest.mark.parametrize('parser', parsers)`, others specifically set `PyYAML`....

Is there rhyme or reason to these choices? Should they be consistent? Or does it matter, much?